sdk/daml-script/test/BUILD.bazel

```diff
@@ -61,18 +63,16 @@ dependencies:
 build-options:
   - --target={target}
 EOF
-      $(location //compiler/damlc) build --project-root=$$TMP_DIR --ghc-option=-Werror -o $$PWD/$(location script-test-v{major}.dar)
+      $(location //compiler/damlc) build --project-root=$$TMP_DIR --ghc-option=-Werror -o $$PWD/$(location script-test.dar)
       rm -rf $$TMP_DIR
     """.format(
-            major = major,
             sdk = sdk_version,
             target = target,
         ),
         tools = ["//compiler/damlc"],
         visibility = ["//visibility:public"],
     )
-    for major in LF_MAJOR_VERSIONS
-    for target in [lf_version_default_or_latest(major)]
+    for target in [lf_version_configuration.get("latest")]
```

Comment:
This doesn't really need to be a loop anymore, are you keeping it with intend to add more cases later?

Reply:
never really needed a loop.
We use the loop comprehension syntax to "emulate" local variable (something like `where target = ...`  in Haskell).